src/Infrastructure/Persistence/Doctrine/Migrations/Version20230928120959.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Persistence\Doctrine\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230928120959 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql("UPDATE vehicle_set SET exempted_types = replace(exempted_types, 'ambulance','emergencyServices') WHERE exempted_types LIKE '%ambulance%'");
```

Comment:
```suggestion
        $this->addSql("UPDATE vehicle_set SET exempted_types = replace(exempted_types, 'ambulance','emergencyServices')");
```

Le replace fait le search également :)